### DISCUSSION TITLE
Ultima Topbar

### DISCUSSION THREAD

**Discussion by erickdeoliveiraleal** [primefaces/community#2360] [PrimeFaces Templates] [answered]
In the Ultima theme, when you click on the search icon in the topbar it opens, but when you click on the X to close, the search box remains open.

**Comment by melloware** [comment, marked as answer]
in `topbar.xhtml` add `type="button"` to the Command Button and it fixes it.

```xml
<p:commandButton icon="pi pi-times" class="rounded-button ui-button-flat ui-button-plain rounded-circle" pt:data-p-styleclass-hide="true" type="button" />
```

**Reply by erickdeoliveiraleal** [reply]
Thanks, I hope they fix it in the package since it's on the theme's production page.

**Comment by tandraschko** [comment]
fixed it in trunk, will be in the next release
